airflow/models/taskinstance.py

```diff
@@ -354,7 +354,7 @@ class TaskInstance(Base, LoggingMixin):
     unixname = Column(String(1000))
     job_id = Column(Integer)
     pool = Column(String(256), nullable=False)
-    pool_slots = Column(Integer, default=1, nullable=False)
+    pool_slots = Column(Integer, default=1, nullable=False, server_default=text("1"))
```

Comment:
Why does it need `text("1")` instead of just `1`

Reply:
It doesn't really need the `text` but needs `"1"` and not `1`